src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/DelegateHelpers.Generated.cs

```diff
@@ -2,10 +2,112 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.ObjectModel;
+using System.Runtime.CompilerServices;
+using System.Dynamic;
+
 namespace System.Linq.Expressions.Compiler
 {
     internal static partial class DelegateHelpers
     {
+#if netstandard17
```

Comment:
Since this class is partial, we could have split this in two and conditionally include the netstandard1.7 side of it.